addon/reducers/index.js

```diff
@@ -0,0 +1,16 @@
+// please write your own index.js that exports legit reducers
+//
+// import { combineReducers } from 'redux';
+// const foobar = (state, action) => {
+//   //reducer code here
+// };
+//
+// export default combineReducers({
+//   foobar
+// })
+
+export default function(combine) {
+  return (state, action) => {
```

Comment:
That seems like a reasonable approach. I've been using `Ember.warn` for those purposes (that way, it's stripped from production builds).

Also, doesn't this reducer completely fail as-is, since there's no combine being passed in? I think this file needs to just export a reducer, like:
```
export default function (state, action) {
 return state;
};
```